java_training/src/com/lftechnology/jan12/WordListManipulator.java

```diff
@@ -0,0 +1,132 @@
+package com.lftechnology.jan12;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This class makes use of the Collection API and has methods to
+ * <ul>
+ * <li>find the duplicates in a word list</li>
+ * <li>find the unique words in a word list</li>
+ * <li>count the number of duplicates</li>
+ * <li>display the word list with duplicates removed</li>
+ * <li>to makes use of the Shuffle algorithm</li>
+ * </ul>
+ * 
+ * @author nerisa
+ * 
+ */
+public class WordListManipulator {
+	private static final Logger LOG = Logger.getLogger(WordListManipulator.class.getName());
+
+	/**
+	 * @author nerisa
+	 * @param listOfWords
+	 *            {@link List} of words from where the duplicates are to be removed
+	 * @return {@link List} of words with duplicates removed
+	 */
+	public Set<String> removeDuplicates(List<String> listOfWords) {
+		return (new HashSet<String>(listOfWords));
+	}
+
+	/**
+	 * @author nerisa
+	 * @param listOfWords
+	 *            {@link List} of words from where the duplicates are to be found
+	 * @return {@link List} of duplicates words
+	 */
+	public Set<String> findDuplicates(List<String> listOfWords) {
+		Set<String> noDuplicates = removeDuplicates(listOfWords);
+		Set<String> duplicates = new HashSet<String>();
```

Comment:
Since this variable does not need to contain any repeating words, I made its type Set.